lib/hls/hls_parser.js

```diff
@@ -801,6 +801,10 @@ shaka.hls.HlsParser = class {
       ignoreManifestTimestampsInSegmentsMode:
         this.config_.hls.ignoreManifestTimestampsInSegmentsMode,
       type: shaka.media.ManifestParser.HLS,
+      serviceDescription: {
```

Comment:
In the dash parser you are returning {} why are you returning maxLatency and maxPlaybackRate here? (same in other parsers)?

Reply:
I have updated the parser to make serviceDescription optional.